cmake/BLTMacros.cmake

```diff
@@ -48,13 +48,15 @@ include(${BLT_ROOT_DIR}/cmake/BLTPrivateMacros.cmake)
 ##
 ## Adds pre-processor definitions to the given target.
 ##
-## Adds pre-processor definitions to a particular. This macro provides very
+## Adds pre-processor definitions to a particular target. This macro provides very
 ## similar functionality to cmake's native "add_definitions" command, but,
 ## it provides more fine-grained scoping for the compile definitions on a
 ## per target basis. Given a list of definitions, e.g., FOO and BAR, this macro
 ## adds compiler definitions to the compiler command for the given target, i.e.,
 ## it will pass -DFOO and -DBAR.
 ##
+## These compile definitions are not inherited to any dependent targets.
```

Comment:
Does it make sense to add an option (or single value arg) to allow the flags to propagate?

Reply:
I think these three macros need some discussion.